agent/consul/fsm/snapshot.go

```diff
@@ -104,3 +83,934 @@ func (s *snapshot) Persist(sink raft.SnapshotSink) error {
 func (s *snapshot) Release() {
 	s.state.Close()
 }
+
+func RegisteredRestorer() map[structs.MessageType]restorer {
+	registry := map[structs.MessageType]restorer{
```

Comment:
Just pushed those changes.

I also thought that it might be better to make a small(ish) change to ``command.go`` by removing the named struct (`entry`) and just making it an anonymous struct. I thought that it would prevent misuse or confusion.